examples/custom_bazel_integration_test/integration_tests/BUILD.bazel

```diff
@@ -0,0 +1,15 @@
+load(
+    "//:custom_bazel_integration_test.bzl", "custom_bazel_integration_test",
+)
+
+sh_binary(
+    name = "workspace_test_runner",
+    srcs = ["workspace_test.sh"],
+)
+
+custom_bazel_integration_test(
```

Comment:
I'm not sure about it, my idea behind picking this case as an example wasn't to show a macro which can pass args through itself and underlying macro to test runner. But to show that having `args` and `data` supported by `bazel_interation_test` enables creation of this kind of macro which can change the way `test_runner` got executed. `custom_integration_test` itself would have troubles with using data dependencies for example because it's executing the test in the custom location.

If you want this example to be more simple and straightforward demonstration of `args` and `data` usage we can change it to workspace with `bazel_integration_test` with passing argument and data and checking the value of args and content of data files is equals to expected values

Reply:
Sorry. I added this comment to the wrong part of the code. I meant to attach it to the `bazel_integration_test` target declaration. I'll continue the thread there.